avocado/core/nrunner.py

```diff
@@ -372,10 +424,12 @@ def _key_val_args_to_kwargs(kwargs):
 
 def runnable_from_args(args):
     decoded_args = [_arg_decode_base64(arg) for arg in args.get('arg', ())]
-    return Runnable(args.get('kind'),
+    rnb = Runnable(args.get('kind'),
```

Comment:
a small suggestion: rename this to a more explicit name.

Reply:
Yep, the more explicit the better.